client/pages/settings/workspace.vue

```diff
@@ -222,6 +331,42 @@ const customDomainsEnabled = computed(
   () => useRuntimeConfig().public.customDomainsEnabled,
 )
 
+const users = ref([])
+const loadingUsers = ref(true)
+const leaveWorkspaceLoadingState = ref(false)
+const newUser = ref("")
+const newUserRole = ref("user")
+const showEditUserModal = ref(false)
+const selectedUser = ref(null)
+const userNewRole = ref("")
+const updatingUserRoleState = ref(false)
+
+const isWorkspaceAdmin = computed(() => {
+  let user = users.value.find((user) => user.id === authStore.user.id)
+  return user && user.pivot.role === "admin"
+})
+
+const rows = computed(() => {
+  return users.value.filter((user) => user.id !== authStore.user.id)
+})
+
+const columns = computed(()=>{
+  if(isWorkspaceAdmin.value){
+    return [
```

Comment:
Use a ternary for this

return [
      { key: 'name', label: 'Name' },
      { key: 'email', label: 'Email' },
      { key: 'pivot.role', label: 'Role' },
      ...(isWorkspaceAdmin.value ? [{ key: 'actions', label: 'Action' }] : [])
    ]